package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "translation_server",
+  "description": "Gulp for assets compilation",
+  "engines": {
+    "node": "6.2.X",
+    "npm": "3.9.X"
+  },
+  "scripts": {
+    "build": "gulp build",
+    "jest": "jest",
+    "ava": "ava",
+    "eslint": "eslint app/react/**",
+    "test": "yarn eslint && yarn jest && yarn ava"
+  },
+  "devDependencies": {
+    "@ava/babel-preset-transform-test-files": "^3.0.0",
+    "babel-eslint": "^7.1.1",
+    "babel-jest": "^19.0.0",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-class-properties": "^6.19.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-es2015-classes": "^6.18.0",
+    "babel-plugin-transform-object-rest-spread": "^6.20.2",
+    "babel-polyfill": "^6.20.0",
+    "babel-preset-es2015-node6": "^0.4.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-1": "^6.16.0",
+    "babel-register": "^6.18.0",
+    "chokidar": "^1.6.1",
+    "enzyme": "^2.7.0",
+    "enzyme-to-json": "^1.4.5",
+    "eslint": "^3.13.1",
+    "eslint-config-airbnb": "^14.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^3.0.2",
+    "eslint-plugin-react": "^6.9.0",
+    "jest": "^19.0.2",
+    "jest-cli": "^19.0.2",
+    "react-addons-test-utils": "^15.4.2",
+    "sinon": "^2.1.0"
+  },
+  "dependencies": {
+    "ava": "0.16.0",
```

Comment:
Listed packages are `devDependencies` I think. And do we need `babelify`, `browserify`, `uglifyify` and `watchify` packages?

```diff 
 +    "ava": "0.16.0",
 +    "babel-loader": "^6.2.10",
 +    "babel-plugin-transform-runtime": "^6.15.0",
 +    "babel-polyfill": "^6.20.0",
 +    "babel-preset-es2015": "^6.18.0",
 +    "babelify": "^6.3.0",
 +    "browserify": "^12.0.1",
 +    "file-loader": "^0.10.1",
 +    "gulp": "^3.9.1",
 +    "gulp-babel": "^5.3.0",
 +    "gulp-eslint": "2.0.0",
 +    "gulp-rename": "^1.2.2",
 +    "gulp-swig": "^0.8.0",
 +    "gulp-uglify": "^1.5.1",
 +    "html-webpack-plugin": "^2.28.0",
 +    "jest-cli": "^19.0.2",
 +    "morgan": "^1.7.0",
 +    "react-addons-perf": "^15.4.2",
 +    "react-dev-utils": "^0.4.2",
 +    "react-hot-loader": "3.0.0-beta.6",
 +    "react-test-renderer": "^15.4.2",
 +    "source-map-support": "^0.4.9",
 +    "uglifyify": "^3.0.1",
 +    "url-loader": "^0.5.8",
 +    "watchify": "^3.7.0",
 +    "weakmap": "0.0.6",
 +    "webpack": "2.2.0",
 +    "webpack-dev-middleware": "^1.9.0",
 +    "webpack-hot-middleware": "^2.17.1",
 +    "webpack-isomorphic-tools": "^2.6.6"